CONTRIBUTING.md

```diff
@@ -21,6 +21,9 @@ to see if someone else has already reported it. If so, feel free to add
 your scenario, or additional information, to the discussion. Or simply
 "subscribe" to it to be notified when it is updated.
 
+Note: Please do not comment on closed issues or PRs, the maintainers prefer
+that you open a new issue instead.
```

Comment:
That sounds a bit harsh. It is fine to comment on recently closed issues, i.e. if there were just closed for the wrong reason.
The important bit is to not comment on older (inactive) issues/PRs.
I think it is better to just say that older issues/PRs are locked, and if you have a similar problem please open a new issue.

Reply:
Fixed as suggested.  Thanks for providing recommended wording.